articles/_posts/2015-02-25-open-source-study.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Open Study"
```

Comment:
I would include "Open Study: A Call for Help" or something about the call to action in the title here